selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/ImportWizardFormTest.java

```diff
@@ -0,0 +1,614 @@
+
+    gitHub.waitAuthorizationPageOpened();
+    gitHub.typeLogin(gitHubUsername);
+    gitHub.typePass(gitHubPassword);
+    gitHub.clickOnSignInButton();
+
+    // authorize on github.com
+    gitHub.waitAuthorizeBtn();
+    gitHub.clickOnAuthorizeBtn();
+    seleniumWebDriver.switchTo().window(ideWin);
+    loader.waitOnClosed();
+    importProject.selectItemInAccountList(
+        gitHubClientService.getName(gitHubUsername, gitHubPassword));
+    importProject.closeWithIcon();
+
+    menu.runCommand(
+        TestMenuCommandsConstants.Workspace.WORKSPACE,
+        TestMenuCommandsConstants.Workspace.IMPORT_PROJECT);
+    importProject.waitMainForm();
+    importProject.selectGitHubSourceItem();
+    importProject.clickLoadRepoBtn();
+    importProject.selectItemInAccountList(
+        gitHubClientService.getName(gitHubUsername, gitHubPassword));
+    importProject.selectProjectByName(currentProjectName);
+
+    importProject.clickImportBtn();
+
+    // check GitHub identity is present in Keycloak account management page
+    if (isMultiuser) {
+      keycloakFederatedIdentitiesPage.open();
+      assertEquals(keycloakFederatedIdentitiesPage.getGitHubIdentityFieldValue(), gitHubUsername);
+    }
+
+    projectWizard.clickSaveButton();
+    projectExplorer.waitItem(currentProjectName);
+  }
+
+  @Test(priority = 1)
+  public void keepDirectoryImportBySshUrlTest() throws Exception {
+    currentProjectName = keepDirectoryRepo.getName() + "Ssh";
+    projectExplorer.waitProjectExplorer();
+
+    makeKeepDirectoryFromGitUrl(
+        keepDirectoryRepo.getSshUrl(), currentProjectName, DIRECTORY_NAME_1);
+    projectExplorer.waitItem(currentProjectName);
+
+    projectExplorer.waitAndSelectItemByName(currentProjectName);
+    projectExplorer.openItemByPath(currentProjectName);
+    loader.waitOnClosed();
+    projectExplorer.waitItemInvisibility(currentProjectName + "/my-webapp");
+    projectExplorer.waitItem(currentProjectName + "/my-lib");
+    expandDirectoryMyLib(currentProjectName);
+  }
+
+  @Test(priority = 2)
+  public void keepDirectoryImportByHttpsUrlTest() throws Exception {
+    currentProjectName = keepDirectoryRepo.getName() + "Https";
+    projectExplorer.waitProjectExplorer();
+
+    makeKeepDirectoryFromGitUrl(
+        keepDirectoryRepo.getHtmlUrl(), currentProjectName, DIRECTORY_NAME_2);
+    projectExplorer.waitItem(currentProjectName);
+
+    projectExplorer.expandPathInProjectExplorerAndOpenFile(
+        currentProjectName + "/my-lib/src/test/java/hello", "SayHelloTest.java");
+
+    editor.waitActive();
+
+    projectExplorer.waitItemInvisibility(currentProjectName + "/my-lib/src/main");
+    projectExplorer.waitItemInvisibility(currentProjectName + "/my-webapp");
+    projectExplorer.openContextMenuByPathSelectedItem(currentProjectName + "/my-lib/src/test");
+    projectExplorer.clickOnItemInContextMenu(GO_INTO);
+    projectExplorer.waitDisappearItemByPath(currentProjectName + "/src/my-lib");
```

Comment:
it's better to divide action part ffrom and verification part of test by empty row to make test scenario more obvious.